docs/execution/WorkflowExecution.md

```diff
@@ -0,0 +1,96 @@
+This section explains at a high level how Cromwell runs a workflow, and how workflow completeness is determined.
+
+In order to run a workflow, Cromwell uses the backends available to it to create jobs and monitor them until they are complete. When no new jobs can or should be started, and all jobs have reached a terminal status (Success, Failure, Aborted), the workflow itself is terminated.
+
+In a ideal situation, all jobs succeed and the workflow succeeds.
+Unfortunately things don't always go as planned. Here is what to expect from Cromwell when things go off rail.
```

Comment:
off rail => off the rails

Reply:
Since the good captain didn't note it, "a ideal" => "an ideal"